src/sql/workbench/parts/profiler/browser/profilerCopyHandler.ts

```diff
@@ -0,0 +1,26 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the Source EULA. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+import * as os from 'os';
+import { IClipboardService } from 'sql/platform/clipboard/common/clipboardService';
+
+export function HandleCopyRequest(clipboardService: IClipboardService, range: Slick.Range, getCellValue: (row, cell) => string): void {
+	if (range) {
+		let results = '';
+		for (let i = range.fromRow; i <= range.toRow; i++) {
```

Comment:
if this loop can take a long time to complete we should show the busy cursor while it's executing.  we have this problem with the results grid.

Reply:
Profiler might not have this problem, for the events table, it only copies the active cell, for the details table, the number of records are the same as the number of columns.